Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.py

```diff
@@ -538,16 +543,39 @@ def fetch_incidents(client, max_results, last_run, first_fetch, filters):
     return next_run, incidents
 
 
+def params_to_filter(severity, resolution_status):
```

Comment:
Put a blank line after each if and else block for better readabillity. It is a small change but makes the code look emensly better:

```
if a:
    do 1

else:
    do 2

```